lib/wasix/src/runners/wasi_common.rs

```diff
@@ -73,7 +73,8 @@ impl CommonWasiOptions {
 
         if self.mounts.iter().all(|m| m.guest != ".") {
             // The user hasn't mounted "." to anything, so let's map it to "/"
-            builder.add_map_dir(".", "/")?;
+            let path = builder.get_current_dir().unwrap_or(PathBuf::from("/"));
```

Comment:
ah, this'll conflict with my PR. Also, I do believe my change (where I put `/` after `.`) will change how this behaves. @maminrayej let's make sure to account for that in whichever PR gets merged second.

Reply:
I believe wasix-libc does maximal matching when trying to find the preopen dir. So even if `/` is before for example `/home`, `/home/foo` should get matched with the `/home` regardless, but yeah agreed will double check.